src/esp/scene/Mp3dSemanticScene.cpp

```diff
@@ -2,6 +2,7 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
+#include "esp/scene/Mp3dSemanticScene.h"
```

Comment:
Rest of the code adopts the style of including the sibling header without the relative path (i.e. only `Mp3dSemanticScene.h` in this case)

Reply:
Fixing and landing